website/blog/2023-09-14-closed-source-does-not-equal-secure/item.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Closed source does not equal secure
```

Comment:
You might be able to do the original suggestion if you enclose in quotation marks:
```suggestion
title: "Closed source does != more secure"
```